deploy/vm/modules/common_setup/main.tf

```
@@ -1,7 +1,15 @@
 # Create a resource group.
+
+resource null_resource "configuration-check" {
```

This looks like it should work, but I just wanted to make sure that it reliably fails

---

It does. It fails early in the deployment stage when the some module calls the common-setup module which is dependent on configuration-check.